src/local_scheduler/local_scheduler_client.cc

```diff
@@ -75,6 +75,7 @@ void local_scheduler_reconstruct_object(LocalSchedulerConnection *conn,
                                         ObjectID object_id) {
   write_message(conn->conn, RECONSTRUCT_OBJECT, sizeof(object_id),
                 (uint8_t *) &object_id);
+  /* TODO(swang): Propagate the error. */
```

Comment:
Agreed. It just came up during testing, so I wanted to add the TODO so I didn't forget.